src/Spd.Resource.Repository/SharedContactFuncs.cs

```diff
@@ -203,7 +203,10 @@ private static contact UpdateExistingContact(contact existingContact, contact ne
         existingContact.spd_mentalhealthcondition = newContact.spd_mentalhealthcondition ?? existingContact.spd_mentalhealthcondition;
         existingContact.spd_peaceofficer = newContact.spd_peaceofficer ?? existingContact.spd_peaceofficer;
         existingContact.spd_peaceofficerother = newContact.spd_peaceofficerother ?? existingContact.spd_peaceofficerother;
-        existingContact.spd_peaceofficerstatus = newContact.spd_peaceofficer == 100000001 ? newContact.spd_peaceofficerstatus : existingContact.spd_peaceofficerstatus;
+        if (newContact.spd_peaceofficer == 100000000 && (newContact.spd_peaceofficerstatus == null) || newContact.spd_peaceofficerstatus == 100000008) 
```

Comment:
It is better to do this in Mapping.cs under contact.

Reply:
It is part of UpdateExistingContact(), how can be moved to mapping?

Reply:
instead of number, I used Enums value now